src/utils/features.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+import featureJson from '@src/feature-flags.json';
+
+const FEATURES = createFeatureFlags(featureJson, {
+  ShowGraphView: false,
+  ShowRightMenu: false,
+  ActionsPaneFooter: false,
+});
```

Comment:
Downside: you have to define it here + once you want to override it you add it to JSON file. (so twice).
Upside: It's completely timed and easier to remove flags.